code/game/machinery/doors/airlock_control.dm

```diff
@@ -263,7 +263,7 @@
 	..()
 
 /obj/machinery/access_button/attack_hand(mob/user)
-	add_fingerprint(usr)
+	src.add_fingerprint(usr)
```

Comment:
This change does exactly nothing.

Reply:
Trust me, after extensive testing I've found that for some fucked up reason it literally doesn't work at all if it doesn't use .src

Reply:
Then something else is horribly, horribly wrong.

Reply:
I know right? Fingerprint code is fuuuucked but I don't know why. I just want to add my feature and GTFOutta it.